tes-adapter/src/main/java/com/epam/pipeline/adaptor/TesAdapterApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.pipeline.adaptor;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TesAdapterApplication {
```

Comment:
What do you think about introducing WebConfiguration class, at least for enable @EnableWebMvc there, don't forget to Import it here
It also is good practice to use `@ComponentScan` to specify package to be scanned

Reply:
fixed